wagtail_content_import/templatetags/wagtailcontentimport_tags.py

```diff
@@ -17,3 +18,11 @@ def wagtailcontentimport_pickerjs(context):
         js_snippets.append(picker.render_js_init(context['request']))
 
     return mark_safe('\n'.join(js_snippets))
+
+@register.inclusion_tag('wagtail_content_import/picker_buttons.html', takes_context=True)
+def wagtailcontentimport_picker_buttons(context):
+    pickers = [fn() for fn in hooks.get_hooks('register_content_import_picker')]
+    context['default_picker'] = pickers[0]
```

Comment:
Might be possible that there are no pickers registered which would cause a crash.

We also probably want to allow picker functions to return `None` as well so apps could decide not to register one

Reply:
I've now added a check for this - thanks for the heads up. I was initially thinking that they should have at least one, but you're right, there's no point not checking.